apps/nuxt3-ssr/components/content/ContentBlockGeneralDesign.vue

```diff
@@ -16,6 +16,13 @@ watch(cohort, setData, {
   immediate: true,
 });
 
+function toSortedString(input: ISortableNameObject[]) {
+  return input
+    .sort((a, b) => a.order - b.order)
```

Comment:
Yes, I only focus now on age group and propose we don't try to generalize right now. 

So the only change i recommend is that we choose another display component to do the rending as I describe as compared to how it is rendered now. And I do understand we might need to provide a parameter to know how deep you want to query the ontology as optimization measure (I would think we could lazy query these ontology widgets in place of rendering instead of trying to add to the root query to make it a bit faster?) 

Meanwhile talking about future generalization: I indeed agree we should think on how to enhance the gql endpoint specifically with ontologies in mind with a suitable graphql fields that can retrieve subtrees etc, so we can do the incremental loading efficiently in the future (so actually, I think actually very gql like instead of un-gql-like) while also having an endpoint that is really focussed on searching in these terms without these joins. We might take close look on how graph databases do that.